HiP-UserStore.Model/Rest/UserArgs2.cs

```diff
@@ -0,0 +1,36 @@
+﻿using PaderbornUniversity.SILab.Hip.UserStore.Model.Entity;
+
+namespace PaderbornUniversity.SILab.Hip.UserStore.Model.Rest
+{
+    public class UserArgs2
+    {
+        public string UserId { get; set; }
+
+        public string Email { set; get; }
+
+        public string FirstName { get; set; }
+
+        public string LastName { get; set; }
+
+        public string Password { get; set; }
+
+        public StudentDetails StudentDetails { get; set; }
```

Comment:
In our Args classes we don't want to have nested objects. The whole object (and therefore multiple properties) will be changed by one property changed event which exactly the behaviour we don't want to have.  My suggestion would be to remove this property and handle it in another way.